lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -4,7 +4,6 @@ class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmC
     expose :resize_confirm, :override_return => nil
     expose :resize_revert,  :override_return => nil
 
-    expose :supports_resize?
```

Comment:
This is exposed so that users can call it from automate scripts, so I don't think we should remove this.

Reply:
agreed, this line says "please provide `Vm#supports_resize?`.

We have 2 options
- expose delegates over to Vm
- define a method here that calls over to vm.

We transitioned from the first option to the second option

Reply:
think maybe we need to use `object_send` here. researching

Reply:
fixed. thanks